packages/rolldown/src/rolldown-binding.wasi-browser.js

```diff
@@ -16,7 +17,6 @@ const __wasi = new __WASI({
   },
 })
 
-const __wasmUrl = new URL('./rolldown-binding.wasm32-wasi.wasm', import.meta.url).href
```

Comment:
Just check, does this affect anything? I saw some change on this line before in other PRs. @sxzz cc